src/kOS/Utilities/SteeringHelper.cs

```diff
@@ -162,31 +167,65 @@ public static Vector3d GetTorque(Vessel vessel, float thrust)
                 var pod = part as CommandPod;
```

Comment:
we dont have CommandPods anymore, they are all ModuleCommand now. This is likely never being hit